src/pages/home/report/ReportActionCompose/SuggestionMention.tsx

```diff
@@ -408,6 +408,7 @@ function SuggestionMention(
     );
 
     const getSuggestions = useCallback(() => suggestionValues.suggestedMentions, [suggestionValues]);
+    const checkIfSuggestionVisible = useCallback(() => isMentionSuggestionsMenuVisible, [isMentionSuggestionsMenuVisible]);
```

Comment:
```suggestion
    const isSuggestionsMenuVisible = useCallback(() => isMentionSuggestionsMenuVisible, [isMentionSuggestionsMenuVisible]);
```

Reply:
As was discussed above, I changed it to `getIsSuggestionsMenuVisible`